### DISCUSSION TITLE
Multilanguage integrated how was in classic shell?

### DISCUSSION THREAD

**Discussion by seba2282** [Open-Shell/Open-Shell-Menu#1733] [General]
Hello all,
Open Shell will have multilanguage integrated? I see with classic shell i can install in Polish language (my country Poland) and others languages. In open shell still haven't that when install. 

Best Regards 😉

**Comment by ge0rdi** [comment]
Most of strings that `Open-Shell` displays (Start menu, explorer, ...) are localized via `*L10N.ini` files. These are part of the repository so if some strings aren't properly localized anyone could submit pull-request to fix it.

The part that is not localized are settings. Because here it works differently and currently needs special localization DLLs that we don't maintain.
You can still download those via [Temporary Translation/Language Solution](https://github.com/Open-Shell/Open-Shell-Menu#temporary-translationlanguage-solution) described on main page.

More on this in #1627.

**Reply by seba2282** [reply]
Ok, I understand, is about DLL. 
But classic shell had Integrated. You don't maintain that concrete from classic shell. But open shell will have his mean translations. Or any plans? Was perfect see installer and all interface in Polish and other languages, which actually open shell don't support.

**Reply by ge0rdi** [reply]
I have no plans to do anything regarding localization in foreseeable future.

Though if someone is willing to help with these localizations we'd definitely appreciate any help.

**Reply by seba2282** [reply]
Thanks for answer.
